Example of the failure:

```
******************** TEST 'Clang :: InstallAPI/diagnostics-dsym.test' FAILED ********************
Exit Code: 1

Command Output (stderr):
--
RUN: at line 4: rm -rf /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp
+ rm -rf /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp
RUN: at line 5: split-file /Volumes/Work/s/w/ir/x/w/llvm-llvm-project/clang/test/InstallAPI/diagnostics-dsym.test /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp
+ split-file /Volumes/Work/s/w/ir/x/w/llvm-llvm-project/clang/test/InstallAPI/diagnostics-dsym.test /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp
RUN: at line 8: /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/bin/clang --target=arm64-apple-macos11 -g -dynamiclib /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.c  -current_version 1 -compatibility_version 1 -L/Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/usr/lib  -save-temps -dynamiclib  -o /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib -install_name /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib
+ /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/bin/clang --target=arm64-apple-macos11 -g -dynamiclib /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.c -current_version 1 -compatibility_version 1 -L/Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/usr/lib -save-temps -dynamiclib -o /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib -install_name /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib
RUN: at line 12: dsymutil /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib -o /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dSYM
+ dsymutil /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib -o /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dSYM
RUN: at line 14: not /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/bin/clang-installapi -x c++ --target=arm64-apple-macos11  -install_name /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib   -current_version 1 -compatibility_version 1  -o /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/output.tbd -dynamiclib  --verify-against=/Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib --dsym=/Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dSYM  --verify-mode=Pedantic 2>&1 | /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/bin/FileCheck /Volumes/Work/s/w/ir/x/w/llvm-llvm-project/clang/test/InstallAPI/diagnostics-dsym.test
+ not /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/bin/clang-installapi -x c++ --target=arm64-apple-macos11 -install_name /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib -current_version 1 -compatibility_version 1 -o /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/output.tbd -dynamiclib --verify-against=/Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dylib --dsym=/Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.dSYM --verify-mode=Pedantic
+ /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/bin/FileCheck /Volumes/Work/s/w/ir/x/w/llvm-llvm-project/clang/test/InstallAPI/diagnostics-dsym.test
/Volumes/Work/s/w/ir/x/w/llvm-llvm-project/clang/test/InstallAPI/diagnostics-dsym.test:23:10: error: CHECK: expected string not found in input
; CHECK: foo.c:1:0: error: no declaration found for exported symbol 'foo' in dynamic library
         ^
<stdin>:3:205: note: scanning from here
/Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.c:5:0: error: no declaration found for exported symbol 'bar' in dynamic library
                                                                                                                                                                                                            ^

Input file: <stdin>
Check file: /Volumes/Work/s/w/ir/x/w/llvm-llvm-project/clang/test/InstallAPI/diagnostics-dsym.test

-dump-input=help explains the following input dump.

Input was:
<<<<<<
          1: warning: violations found for arm64 [-Winstallapi-violation] 
          2: /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.c:1:0: error: no declaration found for exported symbol 'foo' in dynamic library 
          3: /Volumes/Work/s/w/ir/x/w/llvm_build/tools/clang/stage2-bins/tools/clang/test/InstallAPI/Output/diagnostics-dsym.test.tmp/foo.c:5:0: error: no declaration found for exported symbol 'bar' in dynamic library 
check:23                                                                                                                                                                                                                 X error: no match found
>>>>>>

--

********************
```

After this patch. It passes.

--- a/clang/test/InstallAPI/diagnostics-dsym.test
+++ b/clang/test/InstallAPI/diagnostics-dsym.test
@@ -19,8 +19,8 @@
 ; RUN: --verify-mode=Pedantic 2>&1 | FileCheck %s
 
 ; CHECK: violations found for arm64 
-; CHECK: foo.c:5:0: error: no declaration found for exported symbol 'bar' in dynamic library
-; CHECK: foo.c:1:0: error: no declaration found for exported symbol 'foo' in dynamic library
+; CHECK-DAG: foo.c:5:0: error: no declaration found for exported symbol 'bar' in dynamic library
+; CHECK-DAG: foo.c:1:0: error: no declaration found for exported symbol 'foo' in dynamic library
 
 ;--- foo.c
 int foo(void) {

